client/webpack-hot-middleware-client.js

```diff
@@ -1,4 +1,4 @@
-import webpackHotMiddlewareClient from 'webpack-hot-middleware/client?overlay=false&reload=true&path=/_next/webpack-hmr'
+import webpackHotMiddlewareClient from 'webpack-hot-middleware/client?autoConnect=false'
```

Comment:
I think the overlay option needs use here.
See: 
![screen shot 2017-09-10 at 3 47 12 am](https://user-images.githubusercontent.com/50838/30244406-f8fbbbaa-95da-11e7-985d-a6263bb9c7c0.png)

(Try to add a syntax error)